Sources/AppcuesKit/Presentation/Public/Plugins/AppcuesExperiencePluginConfiguration.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 
 /// An object that decodes instances of a plugin configuration from an Experience JSON model.
 @objc
-internal class AppcuesExperiencePluginConfiguration: NSObject {
+public class AppcuesExperiencePluginConfiguration: NSObject {
```

Comment:
cool that this thing just worked in the repurposed usage here

Reply:
Note that I only made public the parts we need (`decode`). If we made traits/actions public again, the `level` and `renderContext` would probably need to be public too.